test/test_Bassa_endpoint.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import requests
+
+headers = {
+'Host': 'localhost:5000',
+'Accept': 'application/json, text/plain, */*',
+'Accept-Language': 'en-US,en;q=0.5',
+'Accept-Encoding': 'gzip, deflate',
+'Referer': 'http://localhost:3000/',
+'Content-Type': 'application/x-www-form-urlencoded',
+'Content-Length': '28',
+'Origin': 'http://localhost:3000',
+'Connection': 'close'
+}
+payload = {''}
+class TestFlaskAPIUsingRequests(unittest.TestCase):
+    def test_api_for_404(self):
+        response = requests.get('http://localhost:5000')
+        print "code:",response.status_code
+        self.assertEqual(response.status_code,404)
+    def test_api_path_for_404(self):
+        response = requests.get('http://localhost:5000/api')
+        print "code:",response.status_code
+        self.assertEqual(response.status_code,404)
+    def test_api_login(self):
```

Comment:
When writing test cases it's good have descriptive function names which describes what the test case tests for.
eg: test_api_login_returns_auth_level(self)

https://stackoverflow.com/questions/1726622/how-should-unit-tests-be-documented

Reply:
done